main.py

```diff
@@ -2,25 +2,24 @@
 
 
 class Card:
-    suits = ["Clubs", "Hearts", "Spades", "Diamonds"]
-
-    values = [None, None, "2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
-
     def __init__(self, v, s):
         self.value = v
         self.suit = s
 
     def __repr__(self):
-        v = self.values[self.value] + " of " + self.suits[self.suit]
-        return v
+        return self.value + " of " + self.suit
 
 
 class Deck:
+    suits = ["Clubs", "Hearts", "Spades", "Diamonds"]
+
+    values = [None, None, "2", "3", "4", "5", "6", "7", "8", "9", "10", "Jack", "Queen", "King", "Ace"]
+
     def __init__(self):
         self.cards = []
         for i in range(2, 15):
             for j in range(4):
-                self.cards.append(Card(i, j))
+                self.cards.append(Card(self.values[i], self.suits[j]))
```

Comment:
I hadn't noticed that yet. That is much cleaner -- a clear benefit of Deck holding the lists.

Reply:
`for suit in self.suits` is what's called a for-each loop. Instead of keeping track of your index, it just gives you the next item in the row. Way way cleaner. Back when I was doing java/c/c++/etc this didn't exist. But now it's my default. Way way cleaner.

Reply:
Ya, I'm familiar with for-each. I was just doing it that weird way, because I had decided to put the lists in Card. It all becomes easier when you just do things the right way. lol

Reply:
That's what code review is for.

Reply:
simplified loops.